credentials/tls/certprovider/pemfile/watcher.go

```diff
@@ -69,11 +69,17 @@ type Options struct {
 // watch the PEM files specified in the passed in options.
 func NewProvider(o Options) (certprovider.Provider, error) {
 	if o.CertFile == "" && o.KeyFile == "" && o.RootFile == "" {
-		return nil, fmt.Errorf("pemfile: NewProvider(%+v) needs at least one credential file to be specified", o)
+		return nil, fmt.Errorf("pemfile: at least one credential file needs to be specified")
 	}
 	if keySpecified, certSpecified := o.KeyFile != "", o.CertFile != ""; keySpecified != certSpecified {
-		return nil, fmt.Errorf("pemfile: NewProvider(%+v) needs private key file and identity cert file to be both specified or not specified", o)
+		return nil, fmt.Errorf("pemfile: private key file and identity cert file should be both specified or not specified")
 	}
```

Comment:
Can we combine this with the other validation of the same options?  And should we be validating the key/cert paths match here, too?

E.g. `func validateOptions(Options) error`

Oh wait, these are different types `pluginConfig` vs. `Options`.  Why are they different types?

Reply:
Hmm .. not sure why I ended up with two of those types. Removed one. 
I also added a comment on the other anonymous struct which is used to unmarshal the JSON onto. That needs to be different because the `refresh_interval` is specified as a `duration proto` in the JSON, while internally we store it as a `time.Duration`.